bot/cogs/help.py

```diff
@@ -0,0 +1,239 @@
+"""Help system.
+
+This help is nearly perfect, with only one warning (access to a protected
+class) and two info (not enough methods at line 16 and too many arguments at
+line 19), under Flake8, Mypy, Pydocstyle and Pylint.
+"""
+
```

Comment:
I'm not really a huge fan of having comments like these on top of files, the fact that this is help system is very obvious from the name and commenting on how it's nearly perfect doesn't really provide any real reason to keep this

Reply:
It's not to have a D100. We need a module docstring

Reply:
That's why printing the License in each file is useful : it gives a module docstring + it is required by the license, in fact :
`The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.`

Reply:
We don't need module docstring as I described in a comment before

Reply:
Well, as you wish. I'll leave this information level thingy